crates/rolldown/tests/esbuild/default/direct_eval_tainting_no_bundle/artifacts.snap

```diff
@@ -1,22 +1,11 @@
 ---
 source: crates/rolldown_testing/src/integration_test.rs
+snapshot_kind: text
 ---
 # warnings
 
 ## EVAL
 
-```text
```

Comment:
This is expected. https://github.com/rolldown/rolldown/blob/2f7cbff55e3be3da26f383215417174e84d2a936/crates/rolldown/tests/esbuild/default/direct_eval_tainting_no_bundle/entry.js#L21-L26